flex-templates/java/regional_dlp_de_identification/pom.xml

```diff
@@ -0,0 +1,481 @@
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.23</version>
+  </parent>
+
+  <groupId>org.apache.beam.samples</groupId>
+  <artifactId>regional-txt-dlp-bq-streaming</artifactId>
+  <version>1.0.0</version>
+
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <beam.version>2.34.0</beam.version>
+
+    <maven-enforcer-plugin.version>3.0.0</maven-enforcer-plugin.version>
+    <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+    <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
+    <maven-exec-plugin.version>3.0.0</maven-exec-plugin.version>
+
+
+    <slf4j.version>1.7.32</slf4j.version>
+    <google.guava.version>29.0-jre</google.guava.version>
+
+    <apache-http-client-v2>1.39.2</apache-http-client-v2>
+    <avro.version>1.8.2</avro.version>
+    <beam.version>2.34.0</beam.version>
+    <beam-vendor-guava.version>0.1</beam-vendor-guava.version>
+    <extra.enforcer.rules.version>1.3</extra.enforcer.rules.version>
+    <bigtable.version>1.15.0</bigtable.version>
+    <bigtable-beam-import.version>1.15.0</bigtable-beam-import.version>
+    <bigquery.version>v2-rev20210410-1.31.0</bigquery.version>
+    <protobuf.version>3.11.1</protobuf.version>
+    <json.version>20200518</json.version>
+    <re2j.version>1.4</re2j.version>
+    <codehaus-jackson.version>1.9.13</codehaus-jackson.version>
+    <commons-csv.version>1.8</commons-csv.version>
+    <commons-text.version>1.9</commons-text.version>
+    <google-cloud-bigquery.version>1.108.0</google-cloud-bigquery.version>
+    <grpc.version>1.13.1</grpc.version>
+    <derby.version>10.14.2.0</derby.version>
+    <commons-codec.version>1.14</commons-codec.version>
+    <commons-io.version>2.4</commons-io.version>
+    <kafka.clients.version>1.0.0</kafka.clients.version>
+    <threetenbp.version>1.4.4</threetenbp.version>
+    <spring.version>5.2.8.RELEASE</spring.version>
+    <autovalue.annotations.version>1.7.4</autovalue.annotations.version>
+    <autovalue.version>1.7.4</autovalue.version>
+    <tensorflow.version>1.15.0</tensorflow.version>
+    <truth.version>1.0.1</truth.version>
+    <dlp.version>2.1.0</dlp.version>
+    <hadoop.version>3.3.1</hadoop.version>
+    <hbase.version>1.4.5</hbase.version>
+    <scassandra.version>1.1.2</scassandra.version>
+    <cassandra.driver.version>3.6.0</cassandra.driver.version>
+    <kms.version>1.40.0</kms.version>
```

Comment:
I don't think we call kms direcly, but dlp might dpeend on it. if so, let's get the latest version